CMakeLists.txt

```diff
@@ -68,8 +81,12 @@ endif()
 
 find_package(CURL REQUIRED)
 
+file(GLOB_RECURSE src_extras "src/extras/**/*.c" "src/extras/**/*.cpp")
+file(GLOB src_bundles_extras "src/bundles/c/extras/*.c")
+
 add_library(tjs STATIC
     src/builtins.c
+    src/cli.c
```

Comment:
You are re-adding the `cli.c` file, but that contains the main function and shouldn't be part of the library.

Reply:
Thanks for the report, it was a regression when I merged some commits from master.